pkg/cmd/openshift-controller-manager/controller_manager.go

```diff
@@ -25,9 +25,6 @@ import (
 	origincontrollers "github.com/openshift/openshift-controller-manager/pkg/cmd/controller"
 	"github.com/openshift/openshift-controller-manager/pkg/cmd/imageformat"
 	"github.com/openshift/openshift-controller-manager/pkg/version"
-
-	// for metrics
-	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus"
```

Comment:
@deads2k was this import used for anything? If so, is there something to replace it?

Reply:
Did a little more research on this one @adambkaplan 

that `_` import was done very early on in the ocm historywith https://github.com/openshift/openshift-controller-manager/commit/b7179b40d0ecbe386d9a8a079fc2224a5505cf62 as part of the `v0.0.0-alpha0` tag back in Nov 2018.

Today, there are absolutely no other references to that package in the repo.

And across the board now all the controllers in OCM are using `github.com/prometheus/client_golang/prometheus`

Assuming @deads2k is spending his cycles elsewhere I think we can assume this change is in the benign / cleanup category, especially given that it no longer exists in k8s/k8s.

Reply:
If it's no longer upstream, then I am going to assume this is a no-op. Good news is that we do have test coverage on the build metrics in our conformance suite, and this is green.